### DISCUSSION TITLE
Building on Windows with vcpkg Debug config

### DISCUSSION THREAD

**Discussion by 2547techno** [Chatterino/chatterino2#4089] [Q&A] [answered]
When going through the build instructions for Windows on vcpkg I try to build with the "Debug" config instead of the "Release" config but get this error when running the compiled .exe file

![image](https://i.imgur.com/THh222R.png)

Compiling with the "Release" config as outlined in the build instructions works just fine.

Any ideas on how I can resolve this?

**Comment by kornes** [comment, marked as answer]
i ran into that problem few months ago and from what i recall from quick look, debug build coudlnt find plugins. 
Workaround for me was to add 
```
[Paths]
Plugins = ./plugins
```
in `resources\qt.conf `
proper fix would require deeper search

**Reply by 2547techno** [reply]
Thanks this worked!

**Reply by 2547techno** [reply]
~Seems I can't see the debug output when running the exe from the terminal. Is there something else I need to change to see this? I remember when I was using Qt Creator I was able to see them.~ https://learn.microsoft.com/en-us/sysinternals/downloads/debugview
